### PR TITLE
PET  FAO-PM

### PR DESCRIPTION
**Pull request by coxipi**
### Pull Request Checklist:
- [ ] This PR addresses an already opened issue (for bug fixes / features)
    - This PR fixes #xyz
- [x] Tests for the changes have been added (for bug fixes / features)
  - [x] (If applicable) Documentation has been added / updated (for bug fixes / features)
- [x] HISTORY.rst has been updated (with summary of main changes)
  - [x] Link to issue (:issue:`number`) and pull request (:pull:`number`) has been added
- [x] The relevant author information has been added to `.zenodo.json`

### What kind of change does this PR introduce?

* New method of computation for potential_evapotranspiration function.

### Does this PR introduce a breaking change?
Yes, more optional parameters in `water_budget` and `potential_evapotranspiration`

### CODE REVIEW

File: HISTORY.rst [hunk 1]
```diff
@@ -10,7 +10,15 @@ New features and enhancements
...
-* New indices for droughts: SPI (standardized precipitations) and SPEI (standardized water budgets) (:issue:`131`, :pull:`1096`)
+* Method ``FAO_PM98`` (based on Penman-Monteith formula) to compute potential evapotranspiration. (:pull:`1122`).
+
+New indicators
+^^^^^^^^^^^^^^
+* Drought-related indicators: SPI (standardized precipitations) and SPEI (standardized water budgets). (:issue:`131`, :pull:`1096`).
+
+Breaking changes
+^^^^^^^^^^^^^^^^
+* ``water_budget`` and ``potential_evapotranspiration`` now accept radiation (rlds, rlus, rsds, rsus), wind (u2), and relative humidity (hurs) as optional parameters. (:pull:`1122`).
```
Comment (aulemahal) [inline HISTORY.rst:21]:
```suggestion
* ``water_budget`` and ``potential_evapotranspiration`` now accept radiation (rlds, rlus, rsds, rsus), wind (sfcWind), and relative humidity (hurs) as optional parameters. (:pull:`1122`).
```
The cmip6 name, or at least the one we've being using, for wind speed is `sfcwind`. And as far as I know, it doesn't prescribe a specific height?

I'll let you change it everywhere, make suggestions for each case would be quite chaotic ;).

File: xclim/indices/helpers.py [hunk 2]
```diff
@@ -362,3 +362,38 @@ def day_lengths(
     ).assign_attrs(units="h")
 
     return day_length_hours
+
+
+def wind_speed_at_2_meters(
```
Comment (aulemahal) [inline xclim/indices/helpers.py:367]: Could we generalize this method to any height (and not only 2 m)?

File: xclim/testing/tests/test_atmos.py [hunk 3]
```diff
@@ -241,13 +242,26 @@ def test_wind_chill_index(atmosds):
...
+        tn = ds.tasmin
+        tx = ds.tasmax
+        tm = ds.tas
+        hurs = ds.hurs
+        rsds = ds.rsds
+        rsus = ds.rsus
+        rlds = ds.rlds
+        rlus = ds.rlus
+        uas, vas = ds.uas, ds.vas
+        us, _ = xci.uas_vas_2_sfcwind(uas, vas)
```
Comment (aulemahal) [inline xclim/testing/tests/test_atmos.py:257]: This is also an indicator : `atmos.wind_speed_from_vector`.

File: xclim/indices/helpers.py (hunk 2 above)
Comment (coxipi) [reply xclim/indices/helpers.py]: Yes, I will do this

File: HISTORY.rst (hunk 1 above)
Comment (coxipi) [reply HISTORY.rst:21]: I think I changed in the code everywhere, forgot the docs it seems. I'll check everywhere thanks.

File: xclim/indices/_conversion.py [hunk 4]
```diff
@@ -1033,6 +1061,7 @@ def potential_evapotranspiration(
     - "hargreaves85" or "HG85", based on [Hargreaves1985]_. Requires tasmin and tasmax, daily [D] freq. (optional: tas can be given in addition of tasmin and tasmax).
     - "mcguinnessbordne05" or "MB05", based on [Tanguy2018]_. Requires tas, daily [D] freq, with latitudes 'lat'.
     - "thornthwaite48" or "TW48", based on [Thornthwaite1948]_. Requires tasmin and tasmax, monthly [MS] or daily [D] freq. (optional: tas can be given instead of tasmin and tasmax).
+    - "allen98" or "FAO_PM98", based on [Allen1998]_. Modification of Penman-Monteith method. Requires tasmin and tasmax, relative humidity, radiation fluxes, wind speed at two meters.
```
Comment (aulemahal) [inline xclim/indices/_conversion.py:1064]:
```suggestion
    - "allen98" or "FAO_PM98", based on [Allen1998]_. Modification of Penman-Monteith method. Requires tasmin and tasmax, relative humidity, radiation fluxe and wind speed (10 m wind will be converted to 2 m).
```

**Review by aulemahal — APPROVED** [review]
Looks good to me!
The build failure is quite weird though. I can take a look at this.

**Review by tlogan2000 — APPROVED** [review]
as discussed yesterday build failure might be related to @Zeitsperre 's changes to doc's references?  Maybe check with him if you are still having trouble

**Comment by Zeitsperre** [comment]
Feel free to merge when ready! Thanks again!

**Comment by coxipi** [comment]
> Feel free to merge when ready! Thanks again!

Did you do something? I only re-ran the tests. Whatever works haha.

**Comment by Zeitsperre** [comment]
> Did you do something? I only re-ran the tests. Whatever works haha.

The failing test was due to a memory issue we have at times on ReadTheDocs since we tend to trigger many builds as well as install all the dependencies to run our notebooks. We're really pushing the memory/processing limit, so things have been breaking more often lately. The issue I referenced above should help deal with it.

**Pull request merged**